File: 1036.c
#include <stdio.h>
#include <math.h>
int main(){
    double A, B, C, delta, quadra, x1, x2;
    scanf("%lf%lf%lf", &A, &B, &C);
    if(A == 0){
        printf("Impossivel calcular\n");}
    if(A > 0){
        delta = pow(B,2)-4*A*C;
        if(delta < 0){
            printf("Impossivel calcular\n");}
        else if(delta > 0) {
            quadra = pow(delta, 0.5);
            x1 = (-B + quadra) / (2 * A);
            if(x1 == 0){
                printf("Impossivel calcular\n");}
            else{
                printf("R1 = %.5lf\n", x1);}
            x2 = (-B - quadra) / (2 * A);
            if(x2 == 0){
            printf("Impossivel calcular\n");}
            else{
                printf("R2 = %.5lf\n", x2);}
        }
        }
}

File: 1036.java
import java.util.Scanner;
import java.util.Locale;
public class Main {
public static void main(String[] args) {
        Locale.setDefault(Locale.US);
        Scanner input = new Scanner(System.in);
        double a = input.nextDouble();
        double b = input.nextDouble();
        double c = input.nextDouble();
        if(a == 0) {
                System.out.println("Impossivel calcular");
        }
        if(a > 0) {
                double delta = Math.pow(b, 2) - 4 * a * c;
                if (delta < 0) {
                        System.out.println("Impossivel calcular");
                } else if (delta > 0) {
                        double quadra = Math.sqrt(delta);
                        double x1 = (-b + quadra) / (2 * a);
                        if (x1 == 0) {
                                System.out.println("Impossivel calcular");
                        } else {
                                System.out.printf("R1 = %.5f%n", x1);
                        }
                        double x2 = (-b - quadra) / (2 * a);
                        if (x2 == 0) {
                                System.out.println("Impossivel calcular");
                        } else {
                                System.out.printf("R2 = %.5f%n", x2);
                        }
                }
        }
}
}
